package/src/components/AutoCompleteInput/AutoCompleteSuggestionList.tsx

```diff
@@ -22,7 +22,7 @@ import { useTheme } from '../../contexts/themeContext/ThemeContext';
 import { FlatList } from '../../native';
 import type { DefaultStreamChatGenerics } from '../../types/types';
 
-const AUTO_COMPLETE_SUGGESTION_LIST_HEADER_HEIGHT = 30;
+const AUTO_COMPLETE_SUGGESTION_LIST_HEADER_HEIGHT = 50;
```

Comment:
why the sudden increase? this could affect small phones.. you would have to ask design team to check if this increase is okay.. its almost 80% more

Reply:
This is because when there is only one item in the list it gets cut off partially hence its improved.

Reply:
Before:
![Simulator Screenshot - iPhone SE (3rd generation) - 2024-02-20 at 12 46 50](https://github.com/GetStream/stream-chat-react-native/assets/39884168/f151f7ca-0d98-4c0d-b67d-13da4821f36a)

After:
![Simulator Screenshot - iPhone SE (3rd generation) - 2024-02-20 at 12 46 31](https://github.com/GetStream/stream-chat-react-native/assets/39884168/ecddee9d-b844-4aaa-9c68-c30d565542d4)